pennylane/_qubit_device.py

```diff
@@ -781,6 +784,18 @@ def statistics(self, observables, shot_range=None, bin_size=None):
                     self.mutual_info(wires0=wires0, wires1=wires1, log_base=obs.log_base)
                 )
 
+            elif obs.return_type is Shadow:
+                if len(observables) > 1:
+                    raise qml.QuantumFunctionError(
+                        "Classical shadows cannot be returned in combination"
+                        " with other return types"
+                    )
```

Comment:
I'm curious, is this because of a fundamental reason, or just to keep the logic simple?

Reply:
It's mainly because the default implementation of classical shadows executes the individual tapes containing the different Pauli measurements in a single shot fashion (similar to what is being done in the demo). Doing this would leave the device in a weird state for any other measurement processes.

Reply:
Marking as resolved due to no follow-up.